lib/cloudregister/smt.py

```diff
@@ -267,6 +273,9 @@ def __request_cert(self):
                             'Attempt %s with %s of %s' % (
                                 attempts, cert_name, retries)
                         )
-                        logging.error('Server %s is unreachable' % ip)
+                        parsed_cert_url = urlparse(cert_url)
+                        logging.error(
+                            'Server %s is unreachable' % parsed_cert_url.netloc
+                        )
```

Comment:
We already know the IP, so no real need to have it parsed via yet another function call.

Reply:
True